packages/auth/jest.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "clearMocks": true,
```

Comment:
In a following PR we can define this as a reusable config in the root level (`jest-internal-js.config.json`)

Reply:
> In a following PR we can define this as a reusable config in the root level (`jest-internal-js.config.json`)

I'm working on this now :sunglasses: